examples/complete/main.tf

```diff
@@ -37,3 +37,36 @@ module "create_scc_instance" {
   en_instance_crn                   = module.event_notification.crn
   skip_cos_iam_authorization_policy = false
 }
+
+locals {
+  scope = [{
+    environment = "ibm-cloud"
+    properties = [
+      {
+        name  = "scope_type"
+        value = "account.resource_group"
+      },
+      {
+        name  = "scope_id"
+        value = module.resource_group.resource_group_id
+      }
+    ]
+  }]
+}
+
+module "create_profile_attachment" {
+  source                 = "../../modules/attachment/"
+  profile_id             = "bfacb71d-4b84-41ac-9825-e8a3a3eb7405" # defaulted to SOC 2
+  scc_instance_id        = module.create_scc_instance.guid
```

Comment:
I think the error you are getting is because the attachment is trying to get created before the COS bucket integration has completed. We could add an explicit `depends_on` here for the whole SCC module, however perhaps we update the outputs of the SCC root module to not output until the `ibm_scc_instance_settings.scc_instance_settings` has completed. We did something similar in base OCP by adding a depends_on in the outputs -> https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/blob/cee78d679c79938901775c3104fc0164afb0ddb7/outputs.tf#L8